django/db/backends/utils.py

```diff
@@ -135,8 +135,10 @@ def typecast_timestamp(s):  # does NOT store time zone information
     # "2005-07-29 09:56:00-05"
     if not s:
         return None
-    if ' ' not in s:
+    if ':' not in s:
```

Comment:
Is this change merely cosmetic? I'm not seeing any test failures if it's reverted. I guess the space would come before a colon so the old version seems more efficient.